Open_Vuln_Score_History/OVSH.py

```diff
@@ -0,0 +1,110 @@
+import requests
+import pandas as pd 
+import json
+import gzip
+import time
+import re
+from tqdm.auto import tqdm
+
+# Small Demo
+RiskToken = "RjUZkf6sir2s4nXyz_zVjAaMf5VQVg9nCokWs6_1xfBQpe7HrUbSr5_m9GkrL-41"
+
+# Setup Data Dump
+headers = {
+    'X-Risk-Token':  RiskToken,
+    'Content-type': 'application/json',
+}
+
+data = '{ "status" : ["open"], "export_settings" : { "format": "json", "model": "vulnerability" } }'
+
+response = requests.post('https://api.kennasecurity.com/data_exports', headers=headers, data=data)
+data = json.loads(response.content)
+search_id = data['search_id']
+
+# Pull Data Dump
+headers = {
+    'X-Risk-Token': RiskToken ,
+}
+
+params = (
+    ('search_id', search_id),
+)
+
+response = requests.get('https://api.kennasecurity.com/data_exports', headers=headers, params=params)
+while True:
+    if (response.status_code != 200):
+        print("Waiting 15 Seconds For Data Dump.")
+        time.sleep(15)
+        response = requests.get('https://api.kennasecurity.com/data_exports', headers=headers, params=params)
+    if (response.status_code == 200):
```

Comment:
Maybe we want this to be the condition of the loop since its when we're interested in breaking?